docs/installation/keycloak/keycloak-minikube.md

```diff
@@ -0,0 +1,178 @@
+# How to Set Up Headlamp in a Minikube Cluster with Keycloak OIDC Authentication
+
+In this tutorial, we'll walk through the process of configuring Headlamp within a Minikube cluster while utilizing Keycloak for OIDC (OpenID Connect) authentication. This tutorial is based on Keycloak version 22.0.4, Minikube version v1.31.2, and Headlamp version 0.20.1.
+
+## Creating a Keycloak Client
+
+To enable OIDC authentication in your Minikube cluster, you'll need to create a Keycloak client. Follow these steps to configure the client in your Keycloak admin panel:
+
+1. Start by accessing your Keycloak admin panel.
+![Keycloak Admin Panel](./keycloak-admin-panel.jpeg)
+
+2. Navigate to the "Clients" section and Click on the "Create client" option.
+![Keycloak Clients Page](./keycloak-clients-list.jpeg)
+
+3. Fill in the "client-id" and "Name" fields and proceed to the next step.
+![Keycloak Create Client](./keycloak-create-client1.jpeg)
+
+4. In the "Capability Config" step, enable "Client authentication" and proceed to the next step.
+![Keycloak Create Client](./keycloak-create-client2.jpeg)
+
+
+6. Add "http://localhost:8000/*" to the "Valid redirect URIs" and save your settings.
+![Keycloak Create Client](./Keycloak-create-client3.jpeg)
+
+
+## Setting up Minikube with the Keycloak OIDC Configuration
+
+To configure Minikube for Keycloak OIDC integration, follow these steps:
+
+1. **Run the following command** to start Minikube with the necessary configuration options:
+
+   ```shell
+   minikube start -p=keycloak \
```

Comment:
I'm not sure about the line break ("\") used here. Because it doesn't work in cmd/powershell.